group_project_v2/stage_components.py

```diff
@@ -718,8 +722,12 @@ class ProjectTeamXBlock(
 
     def student_view(self, context):
         fragment = Fragment()
+        user_details = self.stage.project_api.get_user_details(self.stage.user_id)
+        user_organizations = self.stage.project_api.get_user_organizations(self.stage.user_id)
+        if user_organizations:
+            user_details.organization = user_organizations[0]['display_name']
```

Comment:
@Kelketek duplicates existing [`team_member` property code](https://github.com/open-craft/xblock-group-project-v2/blob/master/group_project_v2/stage.py#L169-L172). Would be great to refactor it to something like

    def get_member_data(user_id):
        # this code

A good place to add it is the lower part of ProjectAPI class, where post-processing methods live (after the TODO "this methods post-process ....")